tests/spec/w3c/headers-spec.js

```diff
@@ -1013,4 +1013,29 @@ describe("W3C — Headers", function() {
       "does not expect this document to become a W3C Recommendation";
     expect(sotdText).toMatch(expectedString);
   });
+  describe("logos", () => {
+    it("adds logos defined by configuration", async () => {
+      const ops = makeStandardOps();
+      const logo = {
+        src: "http://myimage/",
+        alt: "this is a logo",
+        height: 765,
+        width: 346,
+        url: "http://hyperlink/"
+      };
+      const newProps = {
```

Comment:
To clarify my comment from yesterday, we should probably test with at least 2 logos. If possible, we should add an local image here that won't result in a 404, as it makes logs a little less messy.